packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -1023,7 +1023,7 @@ export default function useAutocomplete(props) {
   };
 
   const handleInputMouseDown = (event) => {
-    if (inputValue === '' || !open) {
+    if ((inputValue === '' || !open) && event.buttons === 0) {
```

Comment:
How about?

```suggestion
    if ((inputValue === '' || !open) && event.button === 0) {
```

<img width="797" alt="Screenshot 2023-03-20 at 01 51 17" src="https://user-images.githubusercontent.com/3165635/226222148-a910ba0c-8a90-4436-ace1-c7b1817bde73.png">

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons


We use `event.button` for the select and the slider since #22638

Reply:
Code has been updated @oliviertassinari  thanks for the feedback.

Reply:
@oliviertassinari what's wrong with `event.buttons`, though? They should both work equally fine in this case, no?

Reply:
In the case of `event.buttons`, I imagine that the logic would need to be `event.buttons === 1` while it's `event.button === 0` in our case. But overall, I think that the issue is:

- button:  which button was pressed on the mouse **to** trigger the event
- buttons: which buttons are pressed on the mouse **when** a mouse event is triggered